ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/ntp.j2

```diff
@@ -5,7 +5,12 @@
 {%         if authentication_key.id is arista.avd.defined and
               authentication_key.key is arista.avd.defined and
               authentication_key.hash_algorithm is arista.avd.defined %}
-ntp authentication-key {{ authentication_key.id }} {{ authentication_key.hash_algorithm }} {{ authentication_key.key }}
+{%             set ntp_auth_key_cli = "ntp authentication-key "  ~ authentication_key.id ~ " " ~ authentication_key.hash_algorithm %}
```

Comment:
you have a double space before `~ authentication_key.id`

Reply:
fixed